elasticsearch-model/test/integration/multiple_models_test.rb

```diff
@@ -0,0 +1,156 @@
+require 'test_helper'
+require 'active_record'
+
+Mongo.setup!
+
+module Elasticsearch
+  module Model
+    class MultipleModelsIntegraton < Elasticsearch::Test::IntegrationTestCase
+      context "Multiple models" do
+        setup do
+          ActiveRecord::Schema.define(:version => 1) do
+            create_table :episodes do |t|
+              t.string :name
+              t.datetime :created_at, :default => 'NOW()'
+            end
+
+            create_table :series do |t|
+              t.string :name
+              t.datetime :created_at, :default => 'NOW()'
+            end
+          end
+
+          class ::Episode < ActiveRecord::Base
+            include Elasticsearch::Model
+            include Elasticsearch::Model::Callbacks
+
+            settings index: {number_of_shards: 1, number_of_replicas: 0} do
+              mapping do
+                indexes :name, type: 'string', analyzer: 'snowball'
+                indexes :created_at, type: 'date'
+              end
+            end
+          end
+
+          class ::Series < ActiveRecord::Base
+            include Elasticsearch::Model
+            include Elasticsearch::Model::Callbacks
+
+            settings index: {number_of_shards: 1, number_of_replicas: 0} do
+              mapping do
+                indexes :name, type: 'string', analyzer: 'snowball'
+                indexes :created_at, type: 'date'
+              end
+            end
+          end
+
+          [::Episode, ::Series].each do |model|
+            model.delete_all
+            model.__elasticsearch__.create_index! force: true
+            model.create name: "The #{model.name}"
+            model.create name: "A great #{model.name}"
+            model.create name: "The greatest #{model.name}"
+            model.__elasticsearch__.refresh_index!
+          end
+
+        end
+
+        should "find matching documents across multiple models" do
```

Comment:
Let me check if it's because this latest change: 

https://github.com/miguelff/elasticsearch-rails/commit/9a74f4dec3579ca09e6384e3be5159f009ce081a#diff-bc189b68e631fd3efba47681fce1287eL109